sources/packages/backend/apps/workers/src/controllers/supporting-user/supporting-user.controller.ts

```diff
@@ -52,16 +63,39 @@ export class SupportingUserController {
         jobLogger.log("Supporting users already exists.");
         return job.complete();
       }
-      const supportingUsers =
-        await this.supportingUserService.createSupportingUsers(
-          job.variables.applicationId,
-          job.variables.supportingUsersTypes,
-        );
-      const createdSupportingUsersIds = supportingUsers.map(
-        (supportingUser) => supportingUser.id,
+      const application = await this.applicationService.getApplicationById(
+        job.variables.applicationId,
+      );
+      return this.dataSource.transaction(
+        async (transactionalEntityManager: EntityManager) => {
+          const supportingUsers =
+            await this.supportingUserService.createSupportingUsers(
```

Comment:
Please execute the `createSupportingUsers` using transactional entity manager. Otherwise it will not participate in the transaction.

![image](https://github.com/user-attachments/assets/c0d5f344-0625-4e4e-9291-4e197437abfb)

![image](https://github.com/user-attachments/assets/ad144b24-09e0-43fe-aa93-8c70585c7a80)